hmpps-prisoner-search/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/search/model/PrisonerModelIntTest.kt

```diff
@@ -0,0 +1,46 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.search.model
+
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import uk.gov.justice.digital.hmpps.prisonersearch.search.AbstractSearchIntegrationTest
+import java.time.LocalDate
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class PrisonerModelIntTest : AbstractSearchIntegrationTest() {
```

Comment:
I couldn't find anywhere that tests the basics of loading some data into the Prisoner index and then checking that the same data can be retrieved (for example this checks that the address start date was saved as a date type). Most tests seem to relate to specific index loading or searching functionality.